src/Adapter/Country/ValueObject/Country.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace PrestaShop\PrestaShop\Adapter\Country\ValueObject;
+
+use PrestaShop\PrestaShop\Core\Domain\Country\Exception\CountryConstraintException;
+
+class Country
+{
+    /** @var int */
+    private $id;
```

Comment:
missing space.
also ps uses 
block type for props (not one line)
/**
* @var int
*/

Reply:
Really? :question:  how the block is better than one liner? One liner seems more readable... what am I missing in my understanding?

Reply:
changed it to PS standard as you guys have more experience with PS even though one liner looks better for me

Reply:
I think there is no logical background for that. "It was done like this for years, so we continue doing it" (for the sake of having a uniform codebase)

There are probably pros and cons between these 2 styles but I dont know 😅 I'll just say that using props block types is more common, more standard for this project, so changing that would surprise / create confusion

Reply:
so let's keep props block then to avoid confusion for properties.